workshops/templates/forms/autoupdateprofile.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base_nonav_fixed.html" %}
+
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block content %}
+{% crispy form form_helper %}
+{% endblock %}
+
+{% block extrajs %}
+{{ form.media }}
+{% endblock extrajs %}
```

Comment:
I believe we have a generic form template somewhere. Perhaps you could reuse it.